src/makeCalculator.js

```diff
@@ -37,7 +37,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  Methods.reset();
+
+  return Methods;
 }
 
+const Methods = {
```

Comment:
why upper case? just methods?
```suggestion
const calculator = {
```